src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -211,8 +208,7 @@
     cursor: pointer;
 
     &:hover {
-      z-index: var(--#{$notification-drawer}__list-item--m-hoverable--hover--ZIndex);
-      box-shadow: var(--#{$notification-drawer}__list-item--m-hoverable--hover--BoxShadow);
+      background-color: var(--#{$notification-drawer}__list-item--m-hoverable--hover--BackgroundColor);
```

Comment:
@lboehling would we want the background color to change on focus as well? The outline may not always be easily apparent:

https://github.com/patternfly/patternfly/assets/70952936/a7682311-aa5f-4d2a-8765-8dec6bc5d204

Reply:
@lucia @thatblindgeye I added the hover background to the focus state also, so you can see if you like that.